infra/core/compute/ubuntu-jumpbox.bicep

```diff
@@ -204,29 +168,28 @@ resource virtualMachine 'Microsoft.Compute/virtualMachines@2021-11-01' = {
     osProfile: {
       computerName: computerName
       adminUsername: adminUsername
-      adminPassword: adminPasswordOrKey
-      linuxConfiguration: ((authenticationType == 'password') ? null : linuxConfiguration)
+      adminPassword: adminPassword
     }
     securityProfile: ((securityType == 'TrustedLaunch') ? securityProfileJson : null)
   }
   tags: tags
 }
 
-resource vmExtension 'Microsoft.Compute/virtualMachines/extensions@2022-03-01' = if ((securityType == 'TrustedLaunch') && ((securityProfileJson.uefiSettings.secureBootEnabled == true) && (securityProfileJson.uefiSettings.vTpmEnabled == true))) {
+resource vmExtension 'Microsoft.Compute/virtualMachines/extensions@2023-09-01' = if ((securityType == 'TrustedLaunch') && ((securityProfileJson.uefiSettings.secureBootEnabled == true) && (securityProfileJson.uefiSettings.vTpmEnabled == true))) {
   parent: virtualMachine
-  name: extensionName
+  name: 'GuestAttestation'
   location: location
   properties: {
-    publisher: extensionPublisher
-    type: extensionName
-    typeHandlerVersion: extensionVersion
+    publisher: 'Microsoft.Azure.Security.LinuxAttestation'
+    type: 'GuestAttestation'
+    typeHandlerVersion: '1.0'
     autoUpgradeMinorVersion: true
     enableAutomaticUpgrade: true
     settings: {
       AttestationConfig: {
         MaaSettings: {
-          maaEndpoint: maaEndpoint
-          maaTenantName: maaTenantName
+          maaEndpoint: substring('emptystring', 0, 0)
```

Comment:
Isn't this just '' - can we remove the substring?

Reply:
probably - not sure why it was like that. changing....